app/src/main/java/com/fcaputo/parkingapp/activity/ReservationActivity.kt

```diff
@@ -0,0 +1,20 @@
+package com.fcaputo.parkingapp.activity
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.fcaputo.parkingapp.R
+import com.fcaputo.parkingapp.mvp.model.ReservationModel
+import com.fcaputo.parkingapp.mvp.presenter.ReservationPresenter
+import com.fcaputo.parkingapp.mvp.view.ReservationView
+
+class ReservationActivity : AppCompatActivity()  {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_reservation)
+    }
+
+    override fun onResume() {
+        super.onResume()
+        ReservationPresenter(ReservationModel(), ReservationView(this))
```

Comment:
this should be created on the onCreate method

Reply:
Done